osu.Game.Rulesets.Mania/UI/DrawableManiaJudgement.cs

```diff
@@ -19,22 +20,42 @@ public DrawableManiaJudgement()
         {
         }
 
-        protected override double FadeInDuration => 50;
+        protected override void ApplyMissAnimations()
+        {
+            if (!(JudgementBody.Drawable is DefaultManiaJudgementPiece))
+            {
+                // this is temporary logic until mania's skin transformer returns IAnimatableJudgements
```

Comment:
I suppose that's going to be a separate pull?

Reply:
at some point, not imminently planned though. i'm just fixing osu! rule set for now.